critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
That's just my opinion, but I don't have much stake in this. :) Fork the code if you think that's a good idea.